optimisation/ScanSelectionsTree.py

```diff
@@ -140,23 +200,19 @@
 
 cSignifVsRest, hSignifVsRest, cEstimVsCut, hEstimVsCut = [], [], [], []
 for iPt, (ptMin, ptMax) in enumerate(zip(ptMins, ptMaxs)):
-
```

Comment:
could you please check your editor configuration to check the deletion of blanck lines?